erpnext/manufacturing/doctype/routing/test_routing.py

```diff
@@ -21,6 +21,20 @@ def setUpClass(cls):
 	def tearDownClass(cls):
 		frappe.db.sql('delete from tabBOM where item=%s', cls.item_code)
 
+	def test_update_operation_time_in_linked_boms(self):
+		operations = [{"operation": "Test Operation A", "workstation": "Test Workstation A", "time_in_mins": 30},
+			{"operation": "Test Operation B", "workstation": "Test Workstation A", "time_in_mins": 20}]
+		
+		setup_operations(operations)
+		routing_doc = create_routing(routing_name="Testing Route", operations=operations)
+		bom_doc = setup_bom(item_code=self.item_code, routing=routing_doc.name)
+		self.assertEqual(routing_doc.operations[0].time_in_mins, 30)
+		routing_doc.operations[0].time_in_mins = 90
+		routing_doc.save()
+		self.assertEqual(routing_doc.operations[0].time_in_mins, 90)
+		bom_doc.reload()
+		self.assertEqual(bom_doc.operations[0].time_in_mins, 90)
+
```

Comment:
```suggestion
		routing_doc.operations[0].time_in_mins = 90
		routing_doc.operations[1].time_in_mins = 42.2
		routing_doc.save()
		self.assertEqual(routing_doc.operations[0].time_in_mins, 90)
		self.assertEqual(routing_doc.operations[1].time_in_mins, 42.2)
		bom_doc.reload()
		self.assertEqual(bom_doc.operations[0].time_in_mins, 90)
		self.assertEqual(bom_doc.operations[1].time_in_mins, 42.2)
		
```

Might be better to add check rows other than just the first one. Also, the operations cost update should be checked.